modules/project-comments/client/directives/comment-directives.js

```diff
@@ -165,6 +165,13 @@ angular.module ('comment')
 
           var pagination = tableState.pagination;
           var sort = tableState.sort;
+          var scrollTop = function() {
```

Comment:
Do we need this _scrollTop_ function declared here?
If we can reuse the definition in the controller we would avoid having the same code defined in both places.

Reply:
I couldn't find a different place to put the _scrollTop_ function. If there is a different place to put it we can move it there.

Reply:
I am wondering if it would be worth exposing the function to the global scope, similarly to what is described [here](http://jsfiddle.net/austinnoronha/nukRe/light/) and then just call it as $scope.scrollTop rather than re-declaring it.

Reply:
Yeah sounds good! I'll add the function to the scope.